modules/gitops-release-notes-1-12-2.adoc

```diff
@@ -0,0 +1,79 @@
+
+* Before this update, the `ignoreDifferences` sync option in Argo CD did not work for array fields. This update fixes the issue by modifying the merge strategy of the `ignoreDifferences` sync option used in the upstream project to handle array fields. As a result, the sync option now functions correctly by allowing users to ignore specific elements in the array during sync. link:https://issues.redhat.com/browse/GITOPS-2962[GITOPS-2962]
+
+* Before this update, users were unable to include context for notifications in the `NotificationsConfiguration` custom resource (CR). With this update, users can now add context for notifications by using the `context` field in the `NotificationsConfiguration` CR. This field can be also used to establish shared context across all templates defined in the `NotificationsConfiguration` CR. link:https://issues.redhat.com/browse/GITOPS-4303[GITOPS-4303]
++
+Example of  `context` field in the `NotificationsConfiguration` CR
++
+[source,yaml]
+----
+spec:
+  context:
+    key1: value1
+    key2: value2
+  templates:
+    template.app-created: |-
+      email:
+        subject: Application {{.app.metadata.name}} has been created.
+      message: Application {{.app.metadata.name}} has been created.
+      teams:
+        title: Application {{.app.metadata.name}} has been created.
+    template.app-deleted: |-
+     ...
+     ...
+----
+
+* Before this update, users accessing the ROSA cluster after hibernation were unable to log in to the Argo CD web console due to an error indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors when the ROSA cluster is operational post-hibernation. link:https://issues.redhat.com/browse/GITOPS-4358[GITOPS-4358]
+
+* Before this update, users were unable to log in to the Argo CD web console if the availability of `openshift-gitops` route was delayed while the {gitops-title} Operator processed an Argo CD custom resource instance. An error message was displayed indicating an invalid redirect URI in the Dex configuration. With this update, users can now log in to the Argo CD web console without facing any errors. link:https://issues.redhat.com/browse/GITOPS-3736[GITOPS-3736]
+
+* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases. This update fixes the issue because operator backed resources with the correct versions are included in the `ClusterServiceVersion` manifest file.
```

Comment:
Operator should be capitalized, and I think a hyphen here would be appropriate for the compound adjective:
```suggestion
* Before this update, users could not create custom resources for Argo CD from the *Add* page on the *Developer* perspective of the {gitops-title} Web console. This issue has been observed from {gitops-title} 1.10 and later releases. This update fixes the issue because Operator-backed resources with the correct versions are included in the `ClusterServiceVersion` manifest file.
```

Reply:
Fixed.